<launch>

  <!-- TTS -->
  <node pkg="projectx" type="tts_server.py" name="tts_server" if="$(arg tts)">
    <param name="tts_hiz" value="160" />                            
    <param name="tts_kalinlik" value="40"/>
    <param name="tts_delay" value="6" />
    <param name="tts_quiet" value="0" />
  </node>

</launch>